README.md

```diff
@@ -1 +1,63 @@
-# better-unit
\ No newline at end of file
+# better-unit
+
+better-unit is a wrapper around the [nodeunit](https://www.npmjs.com/package/nodeunit) module, with the following additions:
+
+* Support for comparing `Map`, `Set`, `WeakMap`, and `WeakSet` using the `deepEqual()` assertion.
```

Comment:
Short of that i wonder if its worth publishing this bit and the promise rejection stuff as a nodeunit plugin separate from the promise based api, or maybe just include instructions on how to use just those pieces separate